.github/workflows/dockerhub_deployment.yml

```diff
@@ -0,0 +1,22 @@
+# SPDX-FileCopyrightText: 2021 Alliander N.V.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: Docker Hub Deployment
+
+on: push
+
+jobs:
+  push_to_registry:
+    name: Build and publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Login to Docker Hub
+        run: echo ${{ secrets.DOCKER_PASSWORD }} | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
+      - name: Build and publish Docker Image
+        run: ./gradlew clean build -Pnative -Dquarkus.native.container-build=true -Dquarkus.container-image.build=true -Dquarkus.container-image.group=lfenergycompas -Dquarkus.container-image.name=scl-data-service -Dquarkus.container-image.push=true
```

Comment:
Why not start the name of the container also with compas?
-Dquarkus.container-image.name=compas-scl-data-service

That may be better if someone searches for containers.